backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -159,14 +102,12 @@ public ApiFacility addFacilityNew(
     return new ApiFacility(created);
   }
 
-  public ApiFacility updateFacility(
-      UUID facilityId,
+  public ApiFacility addFacilityNew(
```

Comment:
nit: can you add a comment clarifying the difference between `addFacility` and `addFacilityNew`?

Reply:
great callout by emma, can you add the same comments for `updateFacility` and `updateFacilityNew`. and also add a ticket in the backlog to implement this change so we dont forget about it.